### PR TITLE
Add a bundle package for the ECP proxy applications suite

### PR DESCRIPTION
**Pull request by junghans**
Fix #5646 
Fix #5688

### CODE REVIEW

File: var/spack/repos/builtin/packages/ecp-proxy-apps/package.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    url = 'https://github.com/exascaleproject/proxy-apps/archive/v0.9.tar.gz'
+
+    tags = ['proxy-app', 'ecp-proxy-app']
+
+    version('0.9', '395e9d79ae93e8ad71f1ec9773abdd43')
+
+    depends_on('miniamr@1.0', when='@0.9')
+    depends_on('xsbench@13', when='@0.9')
+
+    # Dummy install for now,  will be removed when metapackage is available
```
Comment (davydden) [inline var/spack/repos/builtin/packages/ecp-proxy-apps/package.py]: any ETA?

Comment (junghans) [reply var/spack/repos/builtin/packages/ecp-proxy-apps/package.py]: Whenever #3133 gets merged ;-)

**Comment by junghans** [comment]
@bhatele : It seems we have to enable travis on @exascaleproject.

File: var/spack/repos/builtin/packages/ecp-proxy-apps/package.py (hunk 1 above)
Comment (davydden) [reply var/spack/repos/builtin/packages/ecp-proxy-apps/package.py]: damn, that's from Feb17 and has conflicts already...

**Comment by junghans** [comment]
@tgamblin @scheibelp @adamjstewart `TEST_SUITE=doc` fails with:
```

reading sources... [100%] workflows
Warning, treated as error:
/home/travis/build/LLNL/spack/lib/spack/docs/package_list.rst:6812:Block quote ends without a blank line; unexpected unindent.
make: *** [html] Error 1
```
Idea?

File: var/spack/repos/builtin/packages/ecp-proxy-apps/package.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+import os
+from spack import *
+
+
+class EcpProxyApps(Package):
+    """ This is a collection of packages that represents the official suite of
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/ecp-proxy-apps/package.py]: I think the space here is kicking off a block quote in RST.  Start the first line right after the triple-quote and start the later lines aligned with the first quote in the triple-quote.

File: var/spack/repos/builtin/packages/ecp-proxy-apps/package.py (hunk 1 above)
Comment (citibeth) [reply var/spack/repos/builtin/packages/ecp-proxy-apps/package.py]: See #7862

**Pull request merged**